ci/clean-up-linux.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -euo pipefail
+IFS=$'\n\t'
+set -x
+
+cd output
+rm -r include
+rm -r share
+find bin -type f -and -not -name m68k-amiga-elf-gdb-add-index | xargs strip
```

Comment:
🤦 Good find. I didn't think of executables outside of the bin directory.

Any ideas on how to use find to find them all, without all the false positives?

I wonder could we just compile everything with `-g0` (don't add debug info) rather than stripping it out at the end?

Reply:
I tried building with -g0 -O2 and cc1 is 26.2 MB rather than 22 MB, so better than 213 MB but not quite the same.

Reply:
https://stackoverflow.com/questions/4458120/search-for-executable-files-using-find-command

Reply:
Sounds like that might still match bash scripts and other executable things that aren't ELFs

I'm thinking of one of two approaches:

1. Dont strip, but compile everything with -O2 and -g0 to optimise and remove debug info; or
2. Try to strip everything, and ignore the return value of strip

What do you think?

Reply:
Yeah, I think strip won't operate on non-ELF files, so just let it run on everything.